docs/scenarios_authoring/AggregatesInTimeWindows.md

```diff
@@ -19,9 +19,9 @@ Nussknacker implements 3 types of time windows - tumbling, sliding and session w
 
 The sliding, tumbling and session window components differ among themselves not only in the way the time window is defined. A second key aspect which differentiates these components is how they `transform' input events into aggregate. This transformation can be either creation of a new (aggregate) event or an enrichment of incoming events with the value of the aggregate.  
 
-A sliding-window in its default configuration (`emitWhenEventLeft` is set to `false`) and a tumbling-window with `On each event` emit mode enrich the incoming event with the value of the aggregate. If there were no aggregates "upstream" which terminated the events which entered scenario, the  events which entered the scenario will still be available downstream. As the variables are "attached" to the events, the `#input` and `#inputMeta` variables will be also available 'downstream'. 
+A sliding-window in its default configuration (`emitWhenEventLeft` is set to `false`) and a window with `On each event` emit mode (either tumbling-window or session-window) enrich the incoming event with the value of the aggregate. If there were no aggregates "upstream" which terminated the events which entered scenario, the  events which entered the scenario will still be available downstream. As the variables are "attached" to the events, the `#input` and `#inputMeta` variables will be also available 'downstream'. 
```

Comment:
May be a bit easier to follow:

and a window with `On each event` emit mode (either tumbling-window or session-window) ->
and both tumbling-window and session-window in `On each event` emit mode

Reply:
changed